pkg/virt-controller/watch/clone/clone.go

```diff
@@ -109,100 +119,102 @@ func (ctrl *VMCloneController) execute(key string) error {
 
 func (ctrl *VMCloneController) sync(vmClone *clonev1alpha1.VirtualMachineClone) (syncInfoType, error) {
 	var syncInfo syncInfoType
+	cloneInfo, err := ctrl.retrieveCloneInfo(vmClone)
+	if err != nil {
+		return syncInfo, err
+	}
+
+	targetType := ctrl.getTargetType(cloneInfo.vmClone)
+
+	switch targetType {
```

Comment:
nit: Assign targetType inline

Reply:
`targetType` is later used at line 145

Reply:
Is the assigned var not available in the `default` case?

Reply:
Done